.travis.yml

```diff
@@ -51,7 +51,9 @@ jobs:
       env: XBUILD=true
     # Doc Site svc-cat.io
     - stage: test
-      script: make docs
+      script:
+      - |
+        make docs
```

Comment:
thanks for reading. sorry for confusion on this line. Please do leave a comment if I have left something unexplained.

The history here is I was using this as a location to insert test commands. It needs to be a multiline yaml for the script to have multiple entries.

I left it like this as it both makes the script segments stand out, and makes it easy to add/remove lines to change what is run.